src/core/public/chrome/ui/header/collapsible_nav_group_enabled.scss

```diff
@@ -1,29 +1,49 @@
 .context-nav-wrapper {
   border: none !important;
 
+  .leftNavGroupTitle {
+    font-weight: 400;
+    font-size: 18px;
```

Comment:
Should we use OUI text component to control the font styles?

Reply:
Agree, removed this customized style